src/main/java/com/redhat/pantheon/servlet/ModuleRevisionUpload.java

```diff
@@ -116,6 +128,14 @@ protected void doRun(SlingHttpServletRequest request, PostResponse response, Lis
 
             request.getResourceResolver().commit();
 
+            if (generateHtml) {
+                if (moduleResource == null) {
```

Comment:
Is this conditional necessary? Why not just use the `module` variable everywhere.